cmd/bots/internal/telegram/handlers/handlers.go

```diff
@@ -154,6 +154,58 @@ func InitTgHandlers(environment *common.TelegramBotEnvironment, requestType chan
 			)
 		}
 		return RemoveNodeHandler(c, environment, requestType, responsePairType, args[0])
+
+	})
+
+	environment.Bot.Handle("/add_alias", func(c tele.Context) error {
+		args := c.Args()
+		if len(args) > 2 {
+			return c.Send(
+				messages.RemovedMoreThanOne,
+				&tele.SendOptions{
+					ParseMode: tele.ModeDefault,
+				},
+			)
+		}
+		if len(args) < 1 {
+			return c.Send(
+				messages.RemovedLessThanOne,
+				&tele.SendOptions{
+					ParseMode: tele.ModeDefault,
+				},
+			)
+		}
+		return UpdateAliasHandler(c, environment, requestType, args[0], args[1])
+	})
+
+	environment.Bot.Handle("/aliases", func(c tele.Context) error {
+		nodes, err := messaging.RequestFullNodesList(requestType, responsePairType, false)
+		if err != nil {
+			environment.Zap.Error("failed to request nodes list buttons", zap.Error(err))
+			return err
+		}
+		additionalNodes, err := messaging.RequestFullNodesList(requestType, responsePairType, true)
+		if err != nil {
+			environment.Zap.Error("failed to request list of specific nodes", zap.Error(err))
+			return err
+		}
```

Comment:
Because we don't always need specific (private) nodes, especially for the analyzing part. They're handled differently and I don't want to mix them with the rest of the nodes